### ISSUE TITLE
Missing Excel data

### ISSUE THREAD

**Issue by ldhagen** [datagovsg-blog/circle-line-analytics#1]
Great article and Jupyter Notebook but the Excel data would make it even more interesting and useful. Any chance it can be added or link placed in ReadMe?

**Comment by lgautier** [comment]
Adding data (URL to the data rather than into the repos if too large) would be really nice. Only keep what is necessary if the raw data has more than what can be shared.

That's the kind of nice data story that could be used in workshop, lectures or to showcase methods (visualization, exploratory data analysis, statistics, etc...)

**Comment by maartenmarx** [comment]
I agree. As A data science teacher I would like to use this nice example, and have my students reproduce the findings. Without data that is impossible.

**Comment by jgtann** [comment]
very much in agreement that the data be made available to all as an excellent use case, to promote use of data in real world problem solving
